marsha/settings.py

```diff
@@ -0,0 +1,99 @@
+"""Django settings for marsha project.
```

Comment:
I think @jmaupetit will propose to use DjangoConfiguration.

Reply:
:wink: you read my mind.

Reply:
If it's your choice I'll follow you. Not fan of this kind of heavy machinery that totally changes how settings are managed. But it's only my opinion.

Reply:
I think for coherence in our projects it would be nice to use it.

Reply:
ok then

Reply:
I will but:

- non standard way of defining settings
- cannot define a settings based on a previous one loaded from the env
- cannot use django-admin.py

Digging into the documentation and the code means I won't use it for any project, but I follow your recommandations